app/src/main/java/com/mapbox/navigation/examples/location/ShowCurrentLocationActivity.kt

```diff
@@ -137,7 +137,7 @@ class ShowCurrentLocationActivity : AppCompatActivity() {
 
     @SuppressLint("MissingPermission")
     private fun initNavigation() {
-        mapboxNavigation = MapboxNavigation(
+        mapboxNavigation = MapboxNavigationProvider.create(
```

Comment:
Instead of `MapboxNavigationProvider` let's change this to use `MapboxNavigationApp.setUp()`. Going forward, we are going to officially use `MapboxNavigationApp` more and more and this might become our preferred way to instantiate `mapboxNavigation` starting `v3.0`. `MapboxNavigationProvider` is something we might remove

cc @RingerJK @Zayankovsky @dzinad

Reply:
Wait but `MapboxNavigationApp` is still `Experimental`. Let's use `MapboxNavigationProvider` for now and I will cut a ticket to use `MapboxNavigationApp` once the `Experimental` annotation is removed. Sorry for the confusion.

Reply:
In public docs the provider is used: https://docs.mapbox.com/android/navigation/guides/get-started/initialization/